source/chapter2-devicetree-basics.rst

```diff
@@ -1251,29 +1251,31 @@ interrupt controller.
       #address-cells = <1>;
       #size-cells = <1>;
 
-      open-pic {
+      open_pic: interrupt-controller@13370000 {
+         reg = <0x13370000 0x100>;
          clock-frequency = <0>;
          interrupt-controller;
          #address-cells = <0>;
          #interrupt-cells = <2>;
       };
 
-      pci {
+      pci: pci-controller@47110000 {
```

Comment:
Yeah, we're not very consistent. It should be 'pci@47110000'.

Reply:
Done :)